### DISCUSSION TITLE
Classroom Announcements for QT Client

### DISCUSSION THREAD

**Discussion by computertech1012** [BearWare/TeamTalk5#1399] [General]
Hi,
In TeamTalk 5 Classic, all classroom announcements are reported, for example, when everyone can or can't transmit voice. I am wondering if this feature can be added as well to the QT client?
Does anyone else think this would be a good idea? Because at the moment QT TeamTalk only reports when you can  and can't transmit voice. But it should report for all classroom changes.
It's also good, so that if you take away someone's voice or video then you will know that it's been removed via a TTS event.
Thanks for reading.

**Comment by lbk2907** [comment]
I agree with this suggestion. I have plan to issue this but is no longer
needed for now since you already ask this here.

**Comment by braillescreen** [comment]
Yes. This would be nice.

**Reply by ChrisDuffley** [reply]
See https://github.com/BearWare/TeamTalk5/issues/1401#issuecomment-1186560606 for agreement.

**Comment by bear101** [comment]
You should be able to test classroom announcements with artifacts from this build: https://github.com/BearWare/TeamTalk5/actions/runs/2924251495

**Comment by bear101** [comment]
You can try out the new class room announcements with artifacts from here: https://github.com/BearWare/TeamTalk5/actions?query=branch%3Aissue-1401

**Comment by lbk2907** [comment]
@bear101
I already test this and it is working fine.
Thanks for this nice feature.
😊
